### PR TITLE
fix: fix regression from #1401 and allow CI test failures to break gitthub workflow

### PR DESCRIPTION
**Pull request by BertKleewein**

### CODE REVIEW

File: lib/client.js [hunk 1]
```diff
@@ -1583,7 +1583,7 @@ MqttClient.prototype._handleAck = function (packet) {
   const that = this
   let err
 
-  if (!cb || cb === nop) {
```
Comment (vishnureddy17) [inline lib/client.js:1586]: Can we get an explanation as to what went wrong here?

File: lib/client.js [hunk 2]
```diff
@@ -1583,7 +1583,18 @@ MqttClient.prototype._handleAck = function (packet) {
...
+  // Checking `!cb` happens to work, but it's not technically "correct".
+  //
+  // Why? This code assumes that "no callback" is the same as that "we're not
+  // waiting for responses" (puback, pubrec, pubcomp, suback, or unsuback).
+  //
+  // It would be better to check `if (!this.outgoing[messageId])` here, but
+  // there's no reason to change it and risk (another) regression.
+  //
+  // The only reason this code works is becaues code in MqttClient.publish,
+  // MqttClinet.subsribe, and MqttClient.unsubscribe ensures that we will
```
Comment (vishnureddy17) [inline lib/client.js:1595]: nit: typo
```suggestion
  // MqttClient.subscribe, and MqttClient.unsubscribe ensures that we will
```

**Review by vishnureddy17 — APPROVED** [review]
looks good, small nit

**Comment by Apollon77** [comment]
Thank you very much! Any ETA when we can expoect a new release?

**Comment by Apollon77** [comment]
Thank you for the info!

**Pull request merged**